Tests/UnitTests/Misc/SandboxEnvironmentDetectorTests.swift

```diff
@@ -16,46 +16,96 @@ import XCTest
 
 @testable import RevenueCat
 
+final class MockLocalReceiptFetcher: LocalReceiptFetcherType {
```

Comment:
This can be `private`. Can you move it to the bottom of the file, since it's just an implementation detail? That way the first thing in the file is the actual tests.